package.json

```diff
@@ -40,28 +40,29 @@
   "homepage": "https://docs.opensea.io/v1.0/reference",
   "dependencies": {
     "bignumber.js": "^4.0.4",
+    "ethereumjs-abi": "git+https://github.com/ProjectWyvern/ethereumjs-abi.git",
     "ethereumjs-util": "^5.1.3",
     "fbemitter": "^2.1.1",
     "isomorphic-unfetch": "^2.1.1",
     "json-loader": "^0.5.7",
     "lodash": "^4.17.4",
-    "query-string": "^6.1.0",
+    "query-string": "6.11.0",
```

Comment:
minor but should we keep the carrots `^` so packages will get minor/patch versions automatically?

Reply:
good catch. we might want to make "~" the default because we've run into people doing breaking changes in their minor versions before. will add